texi2rst.py

```diff
@@ -835,7 +835,7 @@ def previsit_element(self, element):
                 if pre:
                     text = pre.get_first_text()
                     if text:
-                        if text.data.startswith('-'):
+                        if text.data.startswith('-') or text.data.lstrip().startswith('-W'):
```

Comment:
What I meant was to have a single:
`if text.data.lstrip().startswith('-'):`
Does that work, or does it break other things?

We chatted last week about setting up a toolchain where we can see the impact of proposed changes, both on the generated .rst files, and on the generated .html/.pdf/.man artefacts.  Do we need that yet?

Reply:
> What I meant was to have a single:
> `if text.data.lstrip().startswith('-'):`
> Does that work, or does it break other things?

It breaks some other places:
https://github.com/marxin/texi2rst-generated/commit/36d37b5e023b126467ff04181518ba15857988d3

> 
> We chatted last week about setting up a toolchain where we can see the impact of proposed changes, both on the generated .rst files, and on the generated .html/.pdf/.man artefacts. Do we need that yet?

Yes, we definitely need that. I've just sent email about it.